libraries/WiFi/src/WiFiClient.cpp

```diff
@@ -224,6 +224,11 @@ size_t WiFiClient::write(const uint8_t *buf, size_t size)
     return res;
 }
 
+size_t WiFiClient::write_P(PGM_P buf, size_t size)
+{
+    return write(buf, size);
+}
+
```

Comment:
At link time, only the functions which were referenced will be left in the program, if -ffunction-section and -gc-sections options are used.